lib/git-metadata-preprocessor/extension.rb

```diff
@@ -0,0 +1,43 @@
+require 'asciidoctor/extensions' unless RUBY_ENGINE == 'opal'
```

Comment:
I think this line is not needed anymore, can you confirm @mojavelinux  ?

Reply:
It's not needed when using 1.5.6, but we haven't set a min version of Asciidoctor of extensions, so we should keep it and remove them all at once when we do.